src/components/QuestionCard/index.js

```diff
@@ -153,8 +145,12 @@ export default function QuestionComponent({question}) {
 						</>
 					)}
 					<article>
-						{question.answers.map(answer => {
-							return <p key={answer.id}>{answer.answerText}</p>;
+						{answers.map(answer => {
+							if (question.answers.find(Element => Element === answer.id)) {
+								console.log(answer);
```

Comment:
Please remove `console.log`s before merging to main :)

Reply:
Will be done. :) Thanks for all your suggestions and help!